vllm/v1/spec_decode/eagle.py

```diff
@@ -313,7 +313,10 @@ def load_model(self, target_model: nn.Module) -> None:
                 self.model.model.embed_tokens = target_model.model.embed_tokens
         else:
             logger.info("Loading EAGLE LM head weights from the target model.")
-            self.model.lm_head = target_model.lm_head
+            if supports_multimodal(target_model) or hasattr(target_model, 'get_language_model'):
```

Comment:
do we need `hasattr(target_model, 'get_language_model')` when we already check for `supports_multimodal(target_model)`?
 The [doc](https://docs.vllm.ai/en/stable/contributing/model/multimodal.html) says to implement the `get_language_model()` .